release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -87,6 +87,48 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-networking"]
 === Networking
 
+[id="ocp-4-13-networking-metrics"]
+==== Enhancements to networking metrics
+===== egress_ips_rebalance_total
```

Comment:
I think this is confusing for users.
Would it be better to put the full metric name so folks can directly enter this into the OCP console instead of having to build the metric name from this doc?

Metric names are in the following format
```${namespace}_${subsystem}_${name}```

so metric `egress_ips_rebalance_total` will be `ovnkube_master_egress_ips_rebalance_total`